naga/src/ir/mod.rs

```diff
@@ -958,14 +977,30 @@ pub enum Binding {
     /// interpolation must be `Flat`.
     ///
     /// [`Vertex`]: crate::ShaderStage::Vertex
+    /// [`Mesh`]: crate::ShaderStage::Mesh
     /// [`Fragment`]: crate::ShaderStage::Fragment
     Location {
         location: u32,
         interpolation: Option<Interpolation>,
         sampling: Option<Sampling>,
+
         /// Optional `blend_src` index used for dual source blending.
         /// See <https://www.w3.org/TR/WGSL/#attribute-blend_src>
         blend_src: Option<u32>,
+
+        /// Whether the binding is a per-primitive binding for use with mesh shaders.
+        ///
+        /// This must be `true` if this binding is a mesh shader primitive output, or such
+        /// an output's corresponding fragment shader input. It must be `false` otherwise.
+        ///
+        /// A stage's outputs must all have unique `location` numbers, regardless of
+        /// whether they are per-primitive; a mesh shader's per-vertex and per-primitive
+        /// outputs share the same location numbering space.
+        ///
+        /// Per primitive values are not interpolated at all and are not dependent on the
```

Comment:
typo: s/Per primitive/Per-primitive


```suggestion
        /// Per-primitive values are not interpolated at all and are not dependent on the
```

Reply:
suggestion: Add a comma here to let clauses in the sentence breath a bit.


```suggestion
        /// Per primitive values are not interpolated at all, and are not dependent on the
```